source/elements/oneDAL/source/data_management/accessors.rst

```diff
@@ -1,3 +1,85 @@
+.. _accessors:
+
 =========
 Accessors
 =========
+
+This section contain definisions of classes that implement :ref:`accessor`
+concept.
+
+------------------------------
+Requirements on accessor types
+------------------------------
+
+Each implementation of :ref:`accessor` concept in |dal_short_name| shall follow
+next requirements:
+
+1. Define single :term:`format of the data <Data format>` of the
+   accessor. Every single accessor type shall provide only one :term:`format of
+   the data <Data format>` returned from it or put inside it.
+
+2. Allow to make access to at least of one in-memory :ref:`dataset`
+   implementation (like :code:`table` or its sub-types or :ref:`table builders
+   <table-builder>`).
```

Comment:
```suggestion
2. Allow it to have access to at least of one in-memory :ref:`dataset`
   implementation (such as :code:`table`, its sub-types, or :ref:`table builders
   <table-builder>`).
```
allow an accessor? I think it should work without `allow` at all: Have access to...